packages/block-serialization-default-parser/parser.php

```diff
@@ -457,7 +457,7 @@ function add_freeform( $length = null ) {
 	 */
 	function add_inner_block( WP_Block_Parser_Block $block, $token_start, $token_length, $last_offset = null ) {
 		$parent = $this->stack[ count( $this->stack ) - 1 ];
-		$parent->block->innerBlocks[] = $block;
+		$parent->block->innerBlocks[] = (array) $block;
```

Comment:
For folks reviewing: http://php.net/manual/en/language.types.array.php#language.types.array.casting